extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,739 @@
+- [Glossary](#glossary)
+- [Examples](#examples)
+  - [Quadtree with four levels](#quadtree-with-four-levels)
+- [Octree Example](#octree-example)
+- [JSON Schema Reference](#json-schema-reference)
+- [Appendix A: Tree Indexing Formulas](#appendix-a-tree-indexing-formulas)
+  - [Morton Indexing Formulas](#morton-indexing-formulas)
+  - [Availability Formulas](#availability-formulas)
+    - [Tile and Content Availability Formulas](#tile-and-content-availability-formulas)
+    - [Child Subtree Availability Formulas](#child-subtree-availability-formulas)
 
-## Contents
-Contents go here.
 ## Overview
 
-This extension to 3D Tiles enables implicit tiling. 
+>**Implicit tiling** is a alternative method for describing a Cesium 3D Tileset that provides a more succinct representation for large tilesets. It encodes the hierarchical structure of the tileset in binary, as opposed to explicit tiling, which specifies the tile hierarchy in JSON. The Cesium 3D Tiles 1.0 specification only supports explicit tiling, as every tile is listed in the tileset JSON file.
+
+Implicit tiling keeps the tileset JSON file small, which makes loading large tilesets faster. While explicit tiling can be used to represent large datasets, the tileset JSON file grows linearly with the number of tiles. Implicit tiling keeps the tileset JSON file bounded in size.
+
+Implicit tiling also provides a method for accessing tiles by tile coordinates. This allows for abbreviated tree traversal algorithms.
+
+![Explicit vs Implicit Tiling](figures/implicit-vs-explicit.jpg)
+
+For a complete list of terminology used, see the [Glossary](#glossary).
+
+## Use Cases
+
+_This section is non-normative_
+
+Implicit tiling allows Cesium 3D Tiles to support a variety of new use cases.
+
+A key use for implicit tiling is enabling and/or accelerating tree traversal algorithms. For example, Cesium uses a [skip-LOD](https://cesium.com/blog/2017/05/05/skipping-levels-of-detail/) algorithm for faster loading times. Accessing a tile by coordinates is faster than traversing the entire tree. Likewise, raycasting algorithms and GIS algorithms can benefit from the abbreviated tree traversals.
+
+Implicit tiling also allows for better interoperability with existing GIS data formats with implicitly defined tiling schemes. Some examples are:
+
+* [CDB](https://docs.opengeospatial.org/is/15-113r5/15-113r5.html)
+* [S2](http://s2geometry.io/)
+* [WMTS](https://www.ogc.org/standards/wmts)
+* [TMS](https://wiki.osgeo.org/wiki/Tile_Map_Service_Specification)
+
+One new feature implicit tiling enables is procedurally-generated tilesets. Since implicit tiling encodes tile coordinates in URLs (such as `{level}/{x}/{y}/model.gltf`), consider the server that serves these files. Instead of serving static files, a server could extract the tile coordinates from the URL and generate tiles at runtime. This could be useful for making a large procedural terrain dataset without requiring much disk space.
```

Comment:
This paragraph could use some rewording. Also we might want to be careful about talking about template URIs since they are not described until a later section.

Reply:
```suggestion
One new feature implicit tiling enables is procedurally-generated tilesets. Since implicit tiling encodes tile coordinates in URLs using [Template URIs](#template-uris), consider the server that serves these files. Instead of serving static files, a server could extract the tile coordinates from the URL and generate tiles at runtime. This could be useful for making a large procedural terrain dataset without requiring much disk space.
```